### DISCUSSION TITLE
CeRAS Case Study

### DISCUSSION THREAD

**Discussion by ghost** [fast-aircraft-design/FAST-OAD#309] [Q&A] [answered]
A couple of questions have arised when using the notebook "CeRAS Case Study" included in FAST-OAD 1.0:

1. Why a "diversion" route is included in FAST-OAD whereas the CeRAS mission profile just shows the main route (climb+cruise+descent). I think I am missing something here. In addition, I don't understand the "holding" and "taxi_in" phases: why 45 min for the first, why 39443.946m for the second?

2. Just out of curiosity, it would be possible to know where the reference "CSR-01_..._out.csv" files have been obtained from?

Thank you in advance!

**Comment by christophe-david** [comment, marked as answer]
My answers below:

> A couple of questions have arised when using the notebook "CeRAS Case Study" included in FAST-OAD 1.0:
> 
> 1. Why a "diversion" route is included in FAST-OAD whereas the CeRAS mission profile just shows the main route (climb+cruise+descent). I think I am missing something here.

Computing this diversion route is part of computing the fuel reserve, as specified in EU-OPS 1.255. This diversion route simulates the additional flight to an alternate destination. CeRAS report does not show it in its mission profile, but clearly mentions an "alternate distance".

> In addition, I don't understand the "holding" and "taxi_in" phases: why 45 min for the first, why 39443.946m for the second?
Holding simulation is also part of the reserve computation and aims at being consistent with EU-OPS 1.255 _(also take into account that we currently stick to the algorithm of the "legacy" FAST, that looks like a simplified application of EU-OPS 1.255. It may need some updates...)_.

Concerning the taxi-in, please note that the distance you cite is an output of the process, not an input. The input is the taxi-in duration (5 min). 
Yet, indeed, this output is incorrect: it appears the considered speed during taxi-in is the one during holding phase. So, we have a little bug here, because I guess that having a more realistic speed should have some effect on the SFC (though I expect very little change on the global results).

> 2. Just out of curiosity, it would be possible to know where the reference "CSR-01_..._out.csv" files have been obtained from?

As stated at beginning of the notebook, they come from [CeRAS-01 database](https://ceras.ilr.rwth-aachen.de/trac/wiki/CeRAS/AircraftDesigns/CSR01).
